de-lib/src/main/java/org/iplantc/de/apps/integration/client/gin/AppEditorGinModule.java

```diff
@@ -8,10 +8,33 @@
 import org.iplantc.de.apps.integration.client.view.AppIntegrationPalette;
 import org.iplantc.de.apps.integration.client.view.AppsEditorView;
 import org.iplantc.de.apps.integration.client.view.AppsEditorViewImpl;
-import org.iplantc.de.apps.integration.client.view.propertyEditors.*;
+import org.iplantc.de.apps.integration.client.view.deployedComponents.cells.DCNameHyperlinkCell;
```

Comment:
This is a general comment about the UI code (not something I think should have been done in this PR), but as we make changes, we should all feel free to rename anything `*DeployedComponent*` (or `DC`) to `*Tool*`.

"Deployed Component" is our obsolete term for "Tools" 🗝